### DISCUSSION TITLE
问题反馈

### DISCUSSION THREAD

**Discussion by shield-s** [MadMaxChow/VLOOK#132] [General]
1、Typora设置深色后，在选择VLOOK主题，VLOOK主题也会变成深色，但每次关闭重开后又变成浅色；
2、使用VLOOK主题，导出时选择默认Html，文件打不开，一直显示正在准备VLOOK插件；
3、导出Html文件打开，3秒阅读提示感觉有点扰人；
4、导出Html文件打开，左侧导航栏每个菜单图标能否加个hover功能名，有些图标点击后才知道是什么功能；

**Comment by MadMaxChow** [comment]
> 1、Typora设置深色后，在选择VLOOK主题，VLOOK主题也会变成深色，但每次关闭重开后又变成浅色；
VLOOK的主题是自动适配Light/Dark显示，会根据Typora或浏览器的Light/Dark变化

> 2、使用VLOOK主题，导出时选择默认Html，文件打不开，一直显示正在准备VLOOK插件；
你检查一下是否没有正确完成导出配置，按简介中的安装配置步骤完成设置就可以

> 3、导出Html文件打开，3秒阅读提示感觉有点扰人；
这个是可以通过md文件中的yaml进行指定使用何种欢迎模式的，例如不显示。

> 4、导出Html文件打开，左侧导航栏每个菜单图标能否加个hover功能名，有些图标点击后才知道是什么功能；
这是个好主意，会在后续版本中优化

**Comment by shuixingzhili** [comment]
29.0为什么没有owl主题了呢

**Reply by MadMaxChow** [reply]
Owl 主题是 VLOOK 官方文档专用，从 V29.0 开始不再对外提供。如果有需要类似风格主题，可以考虑订阅定制的服务
具体看这里：https://madmaxchow.github.io/VLOOK/vip.html

**Comment by shuixingzhili** [comment]
导出html后为什么所有的图片都没有编码，发给别人都看不到，哪里有这个设置吗。还是其他原因导致的

**Reply by MadMaxChow** [reply]
你说的编码是指哪个部分？方便提供一下截图吗？

**Reply by shuixingzhili** [reply]
图片不是会转换为Base64还是其他的编码写入html文件中吗，结果今天发现他没有转换，所有的图片都没有转换，也就是发给别人别人是看不到图片的

**Reply by shuixingzhili** [reply]
更正一下，导出时图片编码了，有base64编码，但是到别的设备就无法显示，自己的电脑上就可以

**Reply by MadMaxChow** [reply]
一般进行 markdown 文档制作时，是不推荐将图片进行 base64 编码的，一是会增加文件体积（例如：100k的图片编码后会变成133k），二是影响浏览器的处理性能。如果是要以离线方式提供导出后的 HTML 给其他人，一般建议是 HTML 与图片一并以文件夹方式打包发布，或是将图片上传至网络的图床形式。

目前 VLOOK 没有针对图片是 base64 编码形式进行过测试～

**Reply by shuixingzhili** [reply]
谢谢您的解疑，确实Vlook没有这个功能，刚刚发现之前都是其他插件的作用，导致导出时进行base64编码
